8.frontend-react/managerapp/src/App.js

```diff
@@ -1,109 +1,136 @@
-import logo from './logo.svg';
-import './App.css';
-import axios from 'axios';
-import Signup from './Pages/Signup';
-import Login from './Pages/Login';
-import NotFound from './Pages/NotFound/NotFound.component';
-import { useEffect, useState } from 'react';
+import Loading from "./Components/Loading/Loading";
+import { useEffect, useState } from "react";
+import axios from "axios";
+import ClientPortal from "./Pages/clientPortal/ClientPortal";
 
 import {
-
   BrowserRouter as Router,
   Switch,
   Redirect,
   Route,
-  Link
+  Link,
+  useLocation,
 } from "react-router-dom";
+import HomePage from "./Pages/HomePage/HomePage";
+import Users from "./Components/Users/Users.component";
+import Clients from "./Components/Clients/Clients.component";
+import SignIn from "./Pages/SignIn/SignIn";
+
```

Comment:
if you like, you can read this article about how to do that in some other way.

https://www.pluralsight.com/guides/how-to-router-redirect-after-login